core/templates/dev/head/app.js

```diff
@@ -382,13 +380,6 @@ oppia.config(['toastrConfig', function(toastrConfig) {
   });
 }]);
 
-oppia.config(['recorderServiceProvider', function(recorderServiceProvider) {
```

Comment:
@pranavsid98, Why have you removed this?
It's not allowing testserver to do mp3 conversion!